commit/operator/operator.pyxbld

```diff
@@ -10,7 +10,7 @@ from commit.operator import config
 
 def make_ext(modname, pyxfilename):
 
-    if (config.nTHREADS is None or config.nTHREADS < 1 or config.nTHREADS > 255):
+    if (config.nTHREADS is None or config.nTHREADS < 0 or config.nTHREADS > 255):
         raise RuntimeError('config.nTHREADS must be between 1 and 255')
```

Comment:
Change `between 1 and`   --> `between 0 and`

Reply:
Why? This way setting nTHREADS=0 would be accepted, but then the code would crash

Reply:
Yes. I just realized that this function is called only if `config.nTHREADS >= 1` (core.build_operator) 
In this case, the condition in the `if` should be as before

Reply:
If I don't change the condition to be config.nTHREADS < 0 or config.nTHREADS > 255, that raises an error during compilation when the user set number of threads = 0.

Reply:
Is the CPU operator required if  `config.nTHREADS==0`?